src/vs/platform/telemetry/test/common/telemetryLogAppender.test.ts

```diff
@@ -58,7 +58,7 @@ class TestTelemetryLogger extends AbstractLogger implements ILogger {
 	flush(): void { }
 }
 
-class TestTelemetryLoggerService implements ILoggerService {
+export class TestTelemetryLoggerService implements ILoggerService {
```

Comment:
@sandy081 Is there a better just generalized test logger I could use? I felt weird stealing this one but it is a telemetry test for the extension host.

Reply:
You can use [NullLoggerService](https://github.com/microsoft/vscode/blob/c6fa8b7e03da3071f0b90ca2a1c07844c9eab470/src/vs/platform/log/common/log.ts#L610)